x-pack/plugins/security_solution/public/resolver/store/selectors.ts

```diff
@@ -122,3 +132,93 @@ function composeSelectors<OuterState, InnerState, ReturnValue>(
 ): (state: OuterState) => ReturnValue {
   return (state) => secondSelector(selector(state));
 }
+
+const spatiallyIndexedEntities = createSelector(
+  (state: ResolverState) => state,
+  function (state) {
+    const indexedProcessTree = indexedProcessTreeFactory(
+      dataSelectors.graphableProcesses(state.data)
+    );
+    const processNodeViewWidth = 360;
```

Comment:
:red_circle: Let's find some way to share these with the node, maybe export from here and import there.